source/ngf-mtl/impl.mm

```diff
@@ -1147,13 +1182,44 @@ ngf_error ngf_create_render_target(const ngf_render_target_info* info, ngf_rende
     NGF_NOEXCEPT {
   assert(info);
   assert(result);
+
+  bool     is_multisample       = false;
+  uint32_t ncolor_attachments   = 0;
+  uint32_t nresolve_attachments = 0;
+
+  for (uint32_t i = 0; i < info->attachment_descriptions->ndescs; ++i) {
+    if (info->attachment_descriptions->descs[i].sample_count > NGF_SAMPLE_COUNT_1) {
+      is_multisample = true;
+    }
+
+    if (info->attachment_descriptions->descs[i].type == NGF_ATTACHMENT_COLOR) {
+      if (info->attachment_descriptions->descs[i].is_resolve) {
+        ++nresolve_attachments;
+      } else {
+        ++ncolor_attachments;
+      }
+    }
+  }
+
+  if (is_multisample && nresolve_attachments == 0) {
```

Comment:
same as for vk, i believe it should be possible to have a multisampled render target that doesn't resolve